tests/primer/primer_tool.py

```diff
@@ -185,15 +185,18 @@ def _create_comment(
             # Create comment for new messages
             count = 1
             fatal_count = 1
+            new_astroid_errors = False
             new_non_fatal_messages = ""
             new_fatal_messages = ""
             if new_messages:
                 print("Now emitted:")
             for message in new_messages:
-                if message["type"] == "fatal":
-                    filepath = str(message["path"]).replace(
-                        str(package_data.clone_directory), ""
-                    )
+                filepath = str(message["path"]).replace(
+                    str(package_data.clone_directory), ""
+                )
+                if message["symbol"] == "astroid-error":
+                    new_astroid_errors = True
+                elif message["type"] == "fatal":
```

Comment:
Shouldn't this be `if`? I would still like these to be reported in the comment, just for us maintainers to notice them.

Reply:
Either way. I thought the idea was to hide as much of this as possible from contributors. We could tag people in the comment. I'm curious what @Pierre-Sassoulas thinks.

Reply:
There's no useful information in the message anyway. All of the useful information is in the GitHub Actions warning link to the crash report.

Reply:
I'm not sure what you're referring to: other fatal errors like `import-error`? We fail the job completely and never get to the comment step.

Reply:
I misunderstood the code 😄 Anyway I think it would be nice to have a little message but to make sure it's clear for contributor that it's a message for maintainer that are going to review, not them.

Reply:
Sure. I'm not sure what else to say, though, other than reporting the issue against astroid. I don't like sending the message that people aren't welcome to help if they want.

Reply:
@DanielNoord Am I correct that after #6746 we won't reach the "fatal errors" block anyway? A few sentences pointing people to the actual info in the logs should be sufficient?

Reply:
I'm talking about non-astroid fatal-errors, like `import-error`.